gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/Manifest.java

```diff
@@ -0,0 +1,117 @@
+package org.apache.gobblin.data.management.copy;
+
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.stream.JsonReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Iterator;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+
+/**
+ * Manifest schema and serDe
+ * https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Manifest+based+distcp+runbook
+ */
+public class Manifest {
+  private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
+  private static final Type CopyableUnitListType = new TypeToken<ArrayList<CopyableUnit>>(){}.getType();
+
+
+  public final ArrayList<CopyableUnit> _copyableUnits;
+
+  public Manifest() {
+    _copyableUnits = new ArrayList<>();
+  }
+
+  public Manifest(ArrayList<CopyableUnit> copyableUnits) {
+    _copyableUnits = copyableUnits;
+  }
+
+  public void add(Manifest.CopyableUnit copyableUnit) {
+    _copyableUnits.add(copyableUnit);
+  }
+
+  public static class CopyableUnit {
+    public final Integer id;
```

Comment:
Let's remove id for now. As it's not needed and might limit the write concurrency, we can easily add it back in the future

Reply:
done